pkgs/by-name/cc/cc1541/package.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromBitbucket
+, asciidoc
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cc1541";
+  version = "4.1";
+
+  src = fetchFromBitbucket {
+    owner = "ptv_claus";
+    repo = "cc1541";
+    rev = version;
+    hash = "sha256-b8cEGC3WxrjebQjed/VD9SIWkiQpNaE2yW+bQRCtmSs=";
+  };
+
+  ENABLE_MAN = 1;
```

Comment:
```suggestion
  env.ENABLE_MAN = 1;
```

Reply:
@OPNA2608  Why bother? Is this not just an aesthetic thing? Does it have any tangible impact?

Reply:
It's not just aesthetic. When using [structuredAttrs](https://github.com/NixOS/nixpkgs/pull/175649) (try opting in with `__structuredAttrs = true`), this doesn't build the documentation - `env.` must be used to set environment variables in Nix.